src/components/Page/index.js

```diff
@@ -34,6 +34,7 @@ const ContentSideNav = dynamic(
   () => import("../core/SideNav").then((mod) => mod.ContentSideNav),
   { ssr: false, loading: () => <p>...</p> }
 );
+import { RATING_POSITION } from "../../constant";
```

Comment:
Can you sort imports up to the file tree? More nested files should be imported after less nested ones.